src/MainWindow.cpp

```diff
@@ -310,13 +313,17 @@ void MainWindow::readSettings()
 
     _statusBarTimeout	 = settings.value( "StatusBarTimeoutMillisec", 3000 ).toInt();
     bool   showTreemap	 = settings.value( "ShowTreemap"	     , true ).toBool();
+    bool   treemapOnSide = settings.value( "TreemapOnSide"	     , true ).toBool();
```

Comment:
See how the '=' signs line up in that entire block? That would be expected here, too.

The whitespace between `bool` and the variable is wrong in the previous line, so it is only consistent that it is wrong here, too. But that was my fault.